src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java

```diff
@@ -3,6 +3,7 @@
 import com.wikia.webdriver.common.contentpatterns.AdsFandomContent;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.JavascriptExecutor'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)